lametro/templates/partials/related_bills.html

```diff
@@ -18,7 +18,7 @@ <h4>Board Reports</h4>
         <td>
           <a href='/board-report/{{ associated_bill.slug }}/' target="_blank">View</a>
         </td>
-        <td><a href={% if report.packet_url %}"{{report.packet_url}}"{% else %}"{{report.url}}"{% endif %}>Download</a></td>
+        <td><a href={% if report.packet.url %}"{{report.packet.url}}"{% else %}"{{report.url}}"{% endif %}>Download</a></td>
```

Comment:
i don't know what the intention of "report.url" was. In django-councilmatic 1.0, bills doe not have a url attributes nor is one defined in la-metro.

Reply:
Looks like it was meant to display document links if a report does not have an associated packet: https://github.com/datamade/la-metro-councilmatic/pull/271